src/controllers/trainee/Controller.ts

```diff
@@ -1,46 +1,44 @@
 import { successHandler } from "../../libs/routes";
 import { Response, Request } from "express";
-
+import * as jwt from "jsonwebtoken";
 class Controller {
   get(req: Request, res: Response) {
     console.log("Inside get");
-    res.send({
-      Name: "Trainee1",
-      message: "get details",
-      status: 200,
-      timestamp: new Date()
-    });
+    const token = req.headers["authorization"];
+    console.log(token);
+    const decode = jwt.verify(token, process.env.KEY);
```

Comment:
please move this to middleware

Reply:
i have used these codes in the controller as a response so that when my authentication is successful the response will be the decoded data.
These codes here have no use but just to give a proper response i used them

Reply:
fixed it and pushed it on feature/40010 branch